ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/INBAND_MGMT_TESTS.yml

```diff
@@ -43,6 +45,8 @@ spine:
   defaults:
     loopback_ipv4_pool: 10.0.254.0/24
     bgp_as: 64999
+    # Dummy IP to avoid errors being raised because of dns_settings for device with oob management.
+    mgmt_ip: 192.168.100.10/24
```

Comment:
Should this be a TODO? this feels clunky - how would a user know?

Reply:
If they use dns_settings it will tell them.

Reply:
well yeah but the fact that it is dummy is confusing. As a user I would be quite surprised to have to give a fake input to make things work.

Reply:
moving to individual devices.